xbmc/addons/AddonDatabase.cpp

```diff
@@ -634,14 +583,14 @@ bool CAddonDatabase::GetAddon(const std::string& addonID, const AddonVersion& ve
     if (!m_pDS)
       return false;
 
-  std::string sql = PrepareSQL(
-      "SELECT addons.id FROM addons "
-      "JOIN addonlinkrepo ON addonlinkrepo.idAddon=addons.id "
-      "JOIN repo ON repo.id=addonlinkrepo.idRepo "
-      "WHERE addons.addonID='%s' AND addons.version='%s' AND repo.addonID='%s'",
-      addonID.c_str(), version.asString().c_str(), repoId.c_str());
+    std::string sql =
```

Comment:
Maybe change direct to `const std::string` as them no more becomes changed and more clear for eyes and maybe compiler.

Reply:
sure, done :+1: